test/system/README.md

```diff
@@ -47,6 +28,9 @@ suggestions or even their implementation.
 
 Make sure you have `bats` and `podman` with `toolbox` installed on your system.
 
+**Important**
+Before you start the tests, you need to have present two images: the default fedora-toolbox image (depending on your version of Fedora) and fedora-toolbox-29
```

Comment:
Rookie mistake: `fedora-toolbox:29`, not `fedora-toolbox-29`. :)

Image names have colon, container names don't.

Reply:
Fixed.